Core/src/game/functions/booleans/is/Is.java

```diff
@@ -868,7 +868,10 @@ public class Is extends BaseBooleanFunction
 		@Opt 	  @Name  final BooleanFunction   contiguous,
              @Opt @Name  final BooleanFunction   If,
  			 @Opt @Name  final BooleanFunction   byLevel,
- 			 @Opt @Name  final BooleanFunction   top
+ 			 @Opt @Name  final BooleanFunction   top,
+ 			 @Opt @Name  final IntFunction   	 throughHowMuch,
+			 @Opt @Name  final BooleanFunction   isVisible,
+			 @Opt @Name  final BooleanFunction   useOpposites

```

Comment:
please update the javadoc comments above this method to also explain the three new params. This javadoc is used to automatically generate the Ludii Language Reference, so it's important to have (even more important than comments are in general)